sdk/jdbc/azure-identity-providers-jdbc/src/main/java/com/azure/identity/providers/jdbc/implementation/token/AccessTokenResolverOptions.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.providers.jdbc.implementation.token;
+
+import com.azure.identity.providers.jdbc.enums.AuthProperty;
+
+import java.util.Properties;
+
+
+/**
+ * Contains details of a request to get a token.
+ */
+public class AccessTokenResolverOptions {
+
+    private String claims;
```

Comment:
make these properties final?

Reply:
these properties are initialized via constructor `AccessTokenResolverOptions(Properties properties)`


```
public AccessTokenResolverOptions(Properties properties) {
        this.tenantId = AuthProperty.TENANT_ID.get(properties);
        this.claims = AuthProperty.CLAIMS.get(properties);

        String scopeProperty = AuthProperty.SCOPES.get(properties);
        this.scopes = scopeProperty == null ?  new String[0] : scopeProperty.split(",");
        this.cacheAccessToken = Boolean.TRUE.equals(AuthProperty.CACHE_ENABLED.getBoolean(properties));
    }
```

if make these properties final, firstly we need to initialize these properties via constructor or other approches .

Reply:
what are you talking about , isn't the., for example, `claims` initialized in the constructor?